.gitignore

```diff
@@ -20,9 +20,7 @@ bin
 *.swp
 
 # envfiles
-.env
 envfile
-.envrc
```

Comment:
Please make these specific to the vendored files, rather than removing the general pickup here